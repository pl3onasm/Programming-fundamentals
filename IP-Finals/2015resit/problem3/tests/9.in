3200 3695
