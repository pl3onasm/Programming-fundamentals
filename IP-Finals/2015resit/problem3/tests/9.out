3203
3221
3251
3257
3271
3299
3301
3319
3343
3347
3359
3371
3373
3389
3391
3407
3433
3463
3467
3469
3511
3527
3541
3571
3583
3613
3643
